### PR TITLE
KYAN-59 add visual tests

### PR DESCRIPTION
**Pull request by Mikilll94**
## Description
- Add visual tests with BackstopJS.
- Remove references to Percy which is not used
- Refine test content used in visual tests

## Motivation and Context
To ensure that that we have no visual regression in the future.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [x] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/Anna Szemiot.png/.content.xml
Comment (paweljozwicki) [inline tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/Anna Szemiot.png/.content.xml:1]: Let's change this asset

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/blog-article-author-bio.png/_jcr_content/renditions/original.png
Comment (paweljozwicki) [inline tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/blog-article-author-bio.png/_jcr_content/renditions/original.png:1]: And this one as well

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/robots.txt/_jcr_content/renditions/original.txt
Comment (paweljozwicki) [inline tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/robots.txt/_jcr_content/renditions/original.txt:1]: I think we can remove robots file

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/sitemap.xml/.content.xml
Comment (paweljozwicki) [inline tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/sitemap.xml/.content.xml:1]: And we could remove sitemap file

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/pages/blog-article-author-bio/.content.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        jcr:lastModified="{Date}2023-06-01T12:21:43.245Z"
+        jcr:lastModifiedBy="wsadmin"
+        jcr:primaryType="ws:PageContent"
+        jcr:title="Blog article author bio"
+        sling:resourceType="kyanite/blogs/components/blogarticlepage"
+        ws:lastModified="{Date}2023-06-02T09:13:21.799Z"
+        ws:lastModifiedBy="wsadmin"
+        ws:template="/libs/kyanite/blogs/templates/blogarticlepage"
+        authorDescription="Goal oriented and not afraid to step out of his comfort zone. He specializes in AEM based websites development and maintenance, including AEM components. Paweł is interested in chess, sports and time with his family."
+        authorName="Anna Szemiot"
```
Comment (paweljozwicki) [inline tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/pages/blog-article-author-bio/.content.xml]: And, let's rename author to John Doe in all the pages?

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/Anna Szemiot.png/.content.xml
Comment (Mikilll94) [reply tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/Anna Szemiot.png/.content.xml:1]: ok, done

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/blog-article-author-bio.png/_jcr_content/renditions/original.png
Comment (Mikilll94) [reply tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/blog-article-author-bio.png/_jcr_content/renditions/original.png:1]: ok, done

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/robots.txt/_jcr_content/renditions/original.txt
Comment (Mikilll94) [reply tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/robots.txt/_jcr_content/renditions/original.txt:1]: ok, done

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/sitemap.xml/.content.xml
Comment (Mikilll94) [reply tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/assets/static/sitemap.xml/.content.xml:1]: ok, done

File: tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/pages/blog-article-author-bio/.content.xml (hunk 1 above)
Comment (Mikilll94) [reply tests/content/src/main/content/jcr_root/content/kyanite-visual-tests/pages/blog-article-author-bio/.content.xml]: okey, done

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**